code/stable/dblpendulum/src/cpp/README.md

```diff
@@ -1,6 +1,8 @@
 # DblPendulum 
 > Author: Dong Chen
 
+
```

Comment:
Something's gone weird: our process shouldn't be adding extra blank lines when there is nothing to show. This is definitely a bug that needs fixed.

Reply:
fixed in 47b7f219572469885d41258ac508cfeae556cbd8, 454828b0ffc1b2bfcdee74d5be725c14dc469bf1, and 3de86113edf2b71994fc0276c1177d81d05170ba